plugins/modules/ios_user.py

```diff
@@ -566,7 +566,7 @@ def main():
         password_type=dict(default="secret", choices=["secret", "password"]),
         privilege=dict(type="int"),
         view=dict(aliases=["role"]),
-        sshkey=dict(type="list", elements="str"),
+        sshkey=dict(type="list", elements="str", no_log=True),
```

Comment:
This is probably not a sensitive attribute since it's the users public key. In such cases we can set `no_log: False` explicitly so that validate-modules is skipped.

Reply:
yea, make sense